src/common/mfu_io.c

```diff
@@ -908,11 +840,14 @@ int daos_mknod(const char* path, mode_t mode, dev_t dev, mfu_file_t* mfu_file)
     else {
         /* create regular file */
         rc = dfs_open(mfu_file->dfs, parent, name,
-                      dfs_mode, O_CREAT,
+                      dfs_mode, O_CREAT | O_EXCL,
                       0, 0, NULL, &(mfu_file->obj));
         if (rc) {
-            MFU_LOG(MFU_LOG_ERR, "dfs_open %s failed (%d %s)",
-                    name, rc, strerror(rc));
+            /* Avoid excessive logging */
+            if (rc != EEXIST) {
+                MFU_LOG(MFU_LOG_ERR, "dfs_open %s failed (%d %s)",
+                        name, rc, strerror(rc));
+            }
```

Comment:
@daltonbohning so are we not printing a message if the file exists already?

Reply:
@dsikich In the code that calls mfu_file_mknod, a message is logged if the file already exists. So the idea here is to just not print what would essentially be a duplicate message.
Once we move these I/O functions into daos, I imagine we might want to remove some of the errors in general, and just return a proper rc/errno and let the caller handle it. I.e. the posix equivalents don't always print messages like this anyway.